setup.py

```diff
@@ -88,7 +89,7 @@
     'deepdiff',
     'nbconvert ==5.3.1',
     'jsonschema ==2.6.0',
-    'cyordereddict',
+    'cyordereddict; python_version < "3"',
```

Comment:
This will not work for our conda based build infrastructure and so we might need a different solution.

Reply:
So, should I add some ``if sys.version_info[0] < 3`` or something of that sort?